engine/src/flutter/shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -473,6 +475,11 @@ @implementation FlutterEngine {
   // Weak reference to last view that received a pointer event. This is used to
   // pair cursor change with a view.
   __weak FlutterView* _lastViewWithPointerEvent;
+
+  // Pointer to a keyboard manager, a hub that manages how key events are
+  // dispatched to various Flutter key responders, and whether the event is
+  // propagated to the next NSResponder.
```

Comment:
Maybe remove the explanation here to `FlutterKeyboardManager`? (I like how you explain it :)

Reply:
That wasn't mine explanation :) I just made property an ivar and move the explanation with it :)

Reply:
Moved.